crates/core/tedge_mapper/src/c8y/dynamic_discovery.rs

```diff
@@ -19,81 +21,54 @@ pub struct DiscoverOp {
 #[derive(thiserror::Error, Debug)]
 #[allow(clippy::enum_variant_names)]
 pub enum DynamicDiscoverOpsError {
-    #[error("Failed to add watch to directory: {0}")]
-    FailedtoAddWatch(String),
-
-    #[error("A non-UTF8 name cannot be used as an operation name: {0:?}")]
-    NotAnOperationName(OsString),
+    #[error("A non-UTF8 path cannot be parsed as an operation: {0:?}")]
+    NotAnOperation(PathBuf),
 
     #[error(transparent)]
     EventError(#[from] std::io::Error),
 }
 
-pub fn create_inotify_watch(ops_dir: PathBuf) -> Result<Inotify, DynamicDiscoverOpsError> {
-    let mut inotify = Inotify::init()?;
-    inotify
-        .add_watch(ops_dir.clone(), WatchMask::CLOSE_WRITE | WatchMask::DELETE)
-        .map_err(|_| {
-            DynamicDiscoverOpsError::FailedtoAddWatch(ops_dir.to_string_lossy().to_string())
-        })?;
-    Ok(inotify)
-}
+/// depending on which editor you use, temporary files could be created.
```

Comment:
Because some editors, if not all, create temporary files in the watched directory and we don't want  to trigger events for those.

@initard Maybe the comment might be improved since not understood by all.